config/config.go

```diff
@@ -68,6 +68,8 @@ func (config *Config) parseFlags() {
 
 	// Sync
 	flag.DurationVar(&config.Sync.Interval, "sync-interval", 15*time.Minute, "Marathon-consul sync interval")
+	flag.BoolVar(&config.Sync.Force, "sync-force", false, "Force leadership-independent Marathon-consul sync")
```

Comment:
OK, I will implement the default hostname resolving but I will have to add `sync-enabled` option as well, as we loose the ability to disable the whole thing now :)